db/migrate/20241206021849_drop_not_null_from_users.rb

```diff
@@ -0,0 +1,6 @@
+class DropNotNullFromUsers < ActiveRecord::Migration[8.0]
+  def change
+    change_column_null :users, :name, true
```

Comment:
Since we might be creating a user record with only an email address if they are added as a collection participant before they've ever logged in, their names will be null.

Reply:
Should we just put the sunetid in as the name so there is some value?

Reply:
That seems a bit confusing to me, but I'm happy to do so if we'd feel better with a value in there rather than allowing null.

Reply:
You can image going to render a list of users and the blanks causing problems.